content/udacity/projects/hosting-full-stack-application.md

```diff
@@ -375,6 +375,78 @@ Create architecture diagrams for an overview of the infrastructure and the pipel
 
 To review the detailed requirements for the project, look at the project [rubric](https://review.udacity.com/#!/rubrics/3070/view).
 
+### Notes if you are using the [Starter code](https://github.com/udacity/nd0067-c4-deployment-process-project-starter)
+
+### Frontend 
+
+
+  1. make sure to change the `apiHost` in the enivronment files to the api link 
+ ```
```

Comment:
need to add the language of the code you write to make sure it was highlighted :) so it will be something like 

````typescript
  apiHost: 'http://example.elasticbeanstalk.com/api/v0'
````